Packs/Code42/Playbooks/playbook-Code42_Create_User_Playbook_README.md

```diff
@@ -0,0 +1,33 @@
+A simple playbook for creating a user in Code42.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* Code42
+
+### Scripts
+This playbook does not use any scripts.
+
+### Commands
+* code42-user-create
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Username | The username to give to the newly created user. |  | Required |
+| Email | The email of the user to create. |  | Required |
+| OrgName | The name of the Code42 organization to create the user in. |  | Required |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Code42 Create User Playbook](Insert the link to your image here)
```

Comment:
`insert the link to you image here` seems like a spot for us to include an image in the repo and link to it